### DISCUSSION TITLE
希望增加只显示译文的模式

### DISCUSSION THREAD

**Discussion by FanXiangTM** [immersive-translate/immersive-translate#434] [Ideas]
### 新功能 | New Feature

虽然沉浸式翻译是该项目的特点，但并不是所有情况都适用，很多时候需要像Chrome自带的网页翻译一样，希望沉浸式翻译能提供这样的选项

### 为什么您想要添加这些功能 | Why you want to add these features

1.
2.
3.

**Comment by versun** [comment]
会做这个功能的哈，关注更新

**Comment by versun** [comment]
0.5.7版本已添加该功能, 火狐浏览器已上架更新, 其他平台的还在审核
点击[更多] -> [切换为仅显示译文]

**Reply by FanXiangTM** [reply]
> 0.5.7版本已添加该功能, 火狐浏览器已上架更新, 其他平台的还在审核 点击[更多] -> [切换为仅显示译文]

Wonderful !
